cmd/kubeadm/app/phases/addons/dns/dns_test.go

```diff
@@ -1414,28 +1414,28 @@ func TestDeployedDNSAddon(t *testing.T) {
 	}{
 		{
 			name:           "default",
-			image:          "registry.k8s.io/coredns/coredns:v1.11.1",
+			image:          "registry.k8s.io/coredns/coredns:v1.11.3",
 			deploymentSize: 1,
-			wantVersion:    "v1.11.1",
+			wantVersion:    "v1.11.3",
 		},
 		{
 			name:           "no dns addon deployment",
-			image:          "registry.k8s.io/coredns/coredns:v1.11.1",
+			image:          "registry.k8s.io/coredns/coredns:v1.11.3",
 			deploymentSize: 0,
 			wantVersion:    "",
 		},
 		{
 			name:           "multiple dns addon deployment",
-			image:          "registry.k8s.io/coredns/coredns:v1.11.1",
+			image:          "registry.k8s.io/coredns/coredns:v1.11.3",
 			deploymentSize: 2,
 			wantVersion:    "",
 			wantErr:        true,
 		},
 		{
 			name:           "with digest",
-			image:          "registry.k8s.io/coredns/coredns:v1.11.1@sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e",
+			image:          "registry.k8s.io/coredns/coredns:v1.11.3@sha256:a0ead06651cf580044aeb0a0feba63591858fb2e43ade8c9dea45a6a89ae7e5e",
```

Comment:
@BenTheElder the SHA for 1.11.3 is different

Reply:
This wasn't the digest for v1.11.1 either, I think it just needs to be a valid digest for parsing / test data, but we can update it.

Reply:
```
$ crane digest registry.k8s.io/coredns/coredns:v1.11.1
sha256:1eeb4c7316bacb1d4c8ead65571cd92dd21e27359f0d4917f1a5822a73b75db1
```

Reply:
I'm not sure if we should do this, it will make updates more challenging and isn't actually necessary for the test here?

I think it would make sense if kubeadm was pinning the digest at runtime to consistently update it, even in test data, but it's deploying by tag, which is why the tests pass without changing the digest.

Reply:
yeah, i don't think it's necessary.